Added: alt attribute to few images (#19842)
mdn/content: files/en-us/learn/server-side/express_nodejs/routes/index.md

- Controller functions to get the requested data from the models, create an HTML page displaying the data, and return it to the user to view in the browser.
- Views (templates) used by the controllers to render the data.

> ![Main data flow diagram of an MVC express server: 'Routes' receive the HTTP requests sent to the Express server and forward them to the appropriate 'controller' function. Controller is used to read and write data from the models. Models are connected to the database to provide data access to the server. Controllers use 'views' to render the data and sends it back to the client as HTTP response.](mvc_express.png)

Reviewer: ```suggestion
![Main data flow diagram of an MVC express server: 'Routes' receive the HTTP requests sent to the Express server and forward them to the appropriate 'controller' function. The controller reads and writes data from the models. Models are connected to the database to provide data access to the server. Controllers use 'views', also called templates, to render the data. The Controller sends the HTML HTTP response back to the client as an HTTP response.](mvc_express.png)
```
